docs/package-management.md

```diff
@@ -0,0 +1,19 @@
+# Package Management in the Monorepo
+
+## Using Local Packages
+
+When working on `diracx-web` or other specified projects, related packages are automatically used from their local versions. This ensures that the latest changes are always in effect during development.
+
+### Configuration
+
+Our monorepo is configured to use workspaces, which allows us to specify the projects and packages that should be linked locally.
+
+**Example `package.json` Configuration:**
+
+```json
+{
+  "workspaces": ["packages/diracx-web-components", "packages/diracx-web"]
+}
+```
+
+Note that any packages that are not listed in the `workspaces` array will install the packages from the npm registry.
```

Comment:
Not crystal clear for someone that is not involved in web development. Could you also expand that part please?